### ISSUE TITLE
minor issues in datagen scripts.

### ISSUE THREAD

**Issue by kevin123456ok** [hci-unihd/MultiStar#1]
Hello,
Thank you for the helpful tool. I've gone through the paper and am eager to train a model with my data. I observed that the scripts in datagen folder can convert Matlab masked data to hdf5, which is quite useful. However, I've encountered a couple of minor issues:
1. All three scripts lack the f.close(), which could lead to the hdf5 file crashing (this occurred on my MacBook).
2. In the ISBI15 script, should the URL be .../~carneiro/... instead of .../~zhi/...?

I would like to know if there is any easy approach to convert imageJ ROI or QuPath annotations to the hdf5 as training data?
